test/fixtures/cookbooks/test/recipes/chefdk.rb

```diff
@@ -2,10 +2,12 @@
   action :install
   channel :stable
   version '0.5.1'
+  omnitruck_endpoint node['chef-ingredient']['omnitruck']['endpoint']
```

Comment:
We would have to do this for every test that supports a configurable omnitruck endpoint.  Trying to think of a better way to do this.